### ISSUE TITLE
Use range context in media queries?

### ISSUE THREAD

**Issue by romainmenke** [allmyfutures/postcss-custom-media-generator#1]
https://github.com/allmyfutures/postcss-custom-media-generator/blob/3fe69171b3bdb06255d0fc9ff45d389fa9c2f474/src/generate_media_queries.ts#L16

This can also be written as :

```
 result[`--${key}-only`] = `(${breakpoint}px <= width < ${nextBreakpoint}px)`;
```

This can avoid issues were browsers produce viewports with fractional pixels.

**Comment by jasonraimondi** [comment]
Thanks for the great tip! 

I didnt think of this because originally, I didn't have this written as an actual plugin, [just a small helper](https://github.com/jasonraimondi/jmondi/tree/d3399109f65f1b764551b5b088552d3a3a4a6f1f/packages/custom-media-mobile-first). I was using the `importFrom` options in the `postcss-custom-media` package, so when that option was removed, I realized I would need to rewrite my helper into an actual plugin.

My one thought on this is that it would then require two plugins to run now, `postcss-custom-media` and `postcss-media-minmax` (or of course, just `postcss-preset-env`), but I do not think that is a problem.

Definitely am going to consider making these tweaks. **Thank you for your input!**
